packages/venia-ui/lib/components/Header/cartTrigger.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { shape, string } from 'prop-types';
 import { ShoppingCart as ShoppingCartIcon } from 'react-feather';
-
+import CREATE_CART_MUTATION from '../../queries/createCart.graphql';
```

Comment:
I bet vscode auto imported this and put it here, but I think we should try to stick to a consistent style of grouping imports going forward (though not documented, and loosly followed). I first saw @supernova-at do this, and really like it. What do you think?

<lib imports>

<@magento imports>

<local imports>